.github/workflows/pip-install-tester.yaml

```diff
@@ -0,0 +1,86 @@
+name: pip install tester
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop, releases/** ]
+  # TODO Replace the workflow_dispatch to schedule
+  # schedule:
+  #   # - cron: '0 16 16 * *'
+  #   - cron: '0 16 * * *'
+
+jobs:
+  pip_test:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
```

Comment:
Can you remove 2.7 and add 3.9 to this list?

Reply:
Yep. I'll do that tomorrow.